### ISSUE TITLE
Open data-file

### ISSUE THREAD

**Issue by iluhaseredkin** [Unidata/netcdf4-python#1013]
Hello! Could you help me? I wanted to use netcdf4-python for data processing https://satdat.ngdc.noaa.gov/sem/poes/data/processed/ngdc/uncorrected/full/2017/metop01/.
I came across an error when reading some data (although, judging by the attributes, it is the same format everywhere). As an example - when reading
'mep_ele_tel90_flux_e1'.
getting a mistake:
"The object of type 'numpy.float64' has no len()."
What's going on inside your library? What could be the problem?

**Comment by jswhit** [comment]
These datasets have `valid_range` set to a scalar 0 instead of a 2-element array.  A workaround is to use `Dataset.set_auto_mask(False)` before reading any data, i.e.

```python
from netCDF4 import Dataset
nc = Dataset('poes_m01_20170101_proc.nc')
nc.set_auto_mask(False)
v = nc['mep_ele_tel90_flux_e1'][:]
```

I will create a PR to fix this in the meantime.

**Comment by jswhit** [comment]
Fix in PR #1013, will be in version 1.5.4 release.

Note that this is really a problem with the netcdf file metadata, but I have modified the library to ignore the invalid `valid_range` instead of raising an exception.

**Comment by iluhaseredkin** [comment]
Hello, Jeff!
Thanks a lot for the prompt reply.

**Comment by jswhit** [comment]
Closed by PR #1014
